Content.Server/Cloning/Components/CloningPodComponent.cs

```diff
@@ -25,6 +27,18 @@ public sealed class CloningPodComponent : Component
         [ViewVariables]
         public bool FailedClone = false;
 
+        /// <summary>
+        /// The material that is used to clone entities.
+        /// </summary>
+        [DataField("requiredMaterial", customTypeSerializer: typeof(PrototypeIdSerializer<MaterialPrototype>)), ViewVariables(VVAccess.ReadWrite)]
+        public string RequiredMaterial = "Biomass";
+
+        /// <summary>
+        /// The entity that is spawned on machine deconstruct as well as failed cloning.
```

Comment:
shouldn't this probably be on material storage so it does the same thing with lathes, etc, with whatever materials are inside it?

Reply:
the main reason it can't work like that is because materials don't have entities tied to them. Implementing it on material storage generally is a good idea though, and I'll do that for a future pr with more material tweaks.

Reply:
i kind of forgot this wasn't merged yet and made a pr that does that